site/src/Librecores/ProjectRepoBundle/Resources/views/Project/view.html.twig

```diff
@@ -195,10 +283,12 @@
 </div>
 {% endblock %}
 
-{% block pagejs %}
-  $(function(){
-    $('[data-toggle="tooltip"]').tooltip();
-    var chart = $('.librecores-project-activity-graph');
-    chart.height('100px').peity('line', { height: chart.height(), width: chart.width() });
-  });
+{% block javascripts %}
+  {{ parent() }}
+  <script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script>
+  {% javascripts
+    'assets/js/metrics.js'
+  filter="?jsqueeze" %}
+  <script src="{{ asset_url }}"></script>
```

Comment:
I assume your goal here is to include the JS only in the pages which actually need it? If that's the goal, please open a issue what we can get this fixed in a more generic way and mark as TODO with the issue number as follow-up.